### PR TITLE
Update striparr.py

### PR DESCRIPTION
**Pull request by townsste**
Should fix  FFmpeg only keeping a single audio & subtitle with conversion.

### CODE REVIEW

**Comment by mikenye** [comment]
Thanks so much for submitting a PR. I've just got a question surrounding syntax - I think the two arguments need to be separate.

**Review by mikenye — APPROVED** [review]

File: rootfs/striparr.py [hunk 1]
```diff
@@ -140,6 +140,7 @@ def worker(filetostrip):
 
                 # add remaining command line bits
                 ffmpeg_strip_command += [
+                     "-map 0",              # keep stream mappings for multiple audio & subtitles
```
[the remark this answers is not in the document]
Comment (townsste) [reply rootfs/striparr.py]: Not sure on that, but i made the change.

[the remark this answers is not in the document]
Comment (mikenye) [reply rootfs/striparr.py]: With `"-map", "0"`, the command line would be `... -map 0 ...`.
With `"-map 0"`, the command line would be `... -map\ 0 ...` (ie, the space part of the argument).

I'm just double-checking this now, will update shortly.

[the remark this answers is not in the document]
Comment (mikenye) [reply rootfs/striparr.py]: We have success!

```
ffmpeg -i /path/to/some/media.mkv -metadata comment= -metadata title= -map 0 -c:v copy -c:a copy -c:s copy /tmp//path/to/some/media.striparrmkv
```

**Comment by mikenye** [comment]
Thanks for your contribution and great job.

**Comment by townsste** [comment]
Small contribution to a great add-on to Radarr and Sonarr.  Thank you for the development and looking into it so quick.

**Comment by townsste** [comment]
Would you able to push a docker update by chance?  I would like to test it.

**Comment by mikenye** [comment]
Done. Let me know how you go.

**Pull request merged**